src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/TableMapping.java

```diff
@@ -290,15 +289,17 @@ private void validateVirtualPhysicalCompatibility() {
     void validateSecondaryIndexes(DynamoTableDescription virtualTable,
                                   DynamoTableDescription physicalTable,
                                   DynamoSecondaryIndexMapper secondaryIndexMapper) {
-        virtualTable.getSis().stream().map(virtualSi -> {
+        List<DynamoSecondaryIndex> secondaryIndexes = virtualTable.getSis();
```

Comment:
could be `final`

Reply:
fixed in db005c1d810368cc2a19533c5362e0ce0f76ebcd